pom.xml

```diff
@@ -41,7 +42,6 @@
         <module>infinispan-client-quickstart</module>
         <module>jms-quickstart</module>
         <module>kafka-quickstart</module>
-        <module>kafka-avro-schema-quickstart</module>
```

Comment:
This doesn't look right.

Reply:
not sure how I managed to do that 🤔 
must have been a bad merge
should be fixed now